app/uk/gov/hmrc/tariffclassificationfrontend/views/components/input_autoComplete.scala.html

```diff
@@ -47,126 +47,8 @@
 </div>
 
 <script>
+        var keywordFieldId = document.getElementById("@{field.id}");
+        var autoCompleteStrings = @Html(Json.toJson(autoCompleteOptions.map{_.replace("'", "&apos;")}).toString);
```

Comment:
We should probably call a function here rather than leaving a variable on the scope and then loading the JS file